src/templates/en/2019/base.html

```diff
@@ -48,7 +49,28 @@
           {{ language_switcher() }}
         </div>
       </nav>
-    </header>
+    </div>
+    {% if request.endpoint == 'home' %}
```

Comment:
Rather than conditionally rendering content specific to the home page, could we overwrite the header block in the index/home template?

Reply:
I guess we could, I must dig a bit more to find out how.

Saw ` if request.endpoint == 'home'` already used for the "Start exploring" button and I thought that is considered good practice in there :)

Reply:
Tried it, apparently, we'd end up with a big chunk of duplicated markup wrapped by `<div class="top-header">`. Unless we isolate it. 

IMHO we could go with the current conditionally rendering for this. In the future, we will reorganize the header with an updated templating logic.